docs/articles/licensing/index.rst

```diff
@@ -1,5 +1,3 @@
-:orphan:
```

Comment:
Hmm should this be removed though?

Reply:
@GordanM , This is no more a single orphan page. It contains "GPLv3 in automotive" page as well as. It enables to access GPLv3 related topics from main menu as well.

Reply:
But will you _always_ have this page included in sub projects inheriting from the blueprint? Ie, there are currently implemented SWF project documentations right now that will likely break if you remove that :orphan: because the page is not included in the toctree anywhere.

Reply:
@frznlogic If that is the case then, the page should be labeled as :orphan: . I will revert that change then. Thanks.